examples/tasks/async_compute.rs

```diff
@@ -0,0 +1,122 @@
+use bevy::prelude::*;
+use bevy::tasks::{AsyncComputeTaskPool, Task};
+use std::time::{Instant, Duration};
+use rand::Rng;
+use std::option::Option::Some;
+use futures_lite::future;
+
+/// This example shows how to use the ecs system and the AsyncComputeTaskPool

```

Comment:
"ECS system" is unclear. Just "ECS" perhaps?

Reply:
changed